grails-app/views/returns/show.gsp

```diff
@@ -278,18 +278,15 @@
                         <tr class="prop">
                             <td class="name">
                                 <g:message code="stockMovement.dateShipped.label"/>
-                                <g:if test="${stockMovement?.shipment?.status?.code==org.pih.warehouse.shipping.ShipmentStatusCode.PENDING}">
-                                    <small><g:message code="default.expected.label"/></small>
-                                </g:if>
                             </td>
                             <td class="value">
-                                <g:if test="${stockMovement?.dateShipped}">
+                                <g:if test="${stockMovement?.shipment?.hasShipped()}">
                                     <span title="${g.formatDate(date:stockMovement?.dateShipped)}">
                                         <g:formatDate format="MMMM dd, yyyy" date="${stockMovement?.dateShipped}"/>
                                     </span>
-                                    <g:if test="${stockMovement?.shipment?.createdBy}">
+                                    <g:if test="${stockMovement?.shipment?.shippedBy || stockMovement?.shipment?.createdBy}">
```

Comment:
Can you move this fallback to created by to the shipment's shippedBy method? It could be something like `if there is no shipped event and there is a case we want to return created by, then it would be returned`.

Reply:
@awalkowiak I guess it should be `if there IS shipped event`, because otherwise we don't want to display any user (until the SM is shipped). 
I'm gonna push the change and let me know if we are on the same page